packages/esm-patient-vitals-app/src/vitals-biometrics-form/vitals-biometrics-form.workspace.tsx

```diff
@@ -164,7 +194,100 @@ const VitalsAndBiometricsForm: React.FC<DefaultPatientWorkspaceProps> = ({
     ],
   );
 
-  const savePatientVitalsAndBiometrics = useCallback(
+  const handleUpdatePatientVitalsAndBiometrics = useCallback(
+    async (formData, abortController, date) => {
+      try {
+        const response = await updatePatientVitalsAndBiometrics(
+          config.concepts,
+          patientUuid,
+          formData,
+          date,
+          abortController,
+          vitalsBiometricsFormData?.uuid,
+          session?.sessionLocation?.uuid,
+        );
+
+        if (response.status === 200) {
```

Comment:
Since this calls `openmrsFetch()` you should not have to do this. The promise will reject if the response is not ok.